src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/RestResponseEntityExceptionHandler.java

```diff
@@ -23,6 +24,14 @@
     private static Logger logger = LoggerFactory.getLogger(ResponseEntityExceptionHandler.class);
 
 
+    @ExceptionHandler(value = {EnhetsregisteretException.class})
+    @ResponseBody
+    protected ResponseEntity<Object> handleEnhetsregisteretException(RuntimeException e, WebRequest webRequest) {
+        return getResponseEntity(e,
+                "Could not get all necessary data for this organization from enhetsregistret",
```

Comment:
Hvorfor engelsk feilmelding? 🤔

Reply:
Ser rart ut ja, men noen andre meldinger er på engelsk ("You don't have access to this ressource", "You are not authorized to access this ressource", ...) trodde det var en konvensjon 🤔
Men jeg endrer denne til norsk allikevel.